### PR TITLE
fix: Updated nextcloud documentation

### PR DESCRIPTION
**Pull request by sjyothi54**
**Description:** Updated the Nextcloud documentation

**Attaching screenshots with changes**
Screenshot 2024-09-13 at 10 55 27 PM
Screenshot 2024-09-13 at 10 55 37 PM

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 1]
```diff
@@ -34,23 +34,23 @@ Complete the following steps to install the integration:
...
+       integrations:
+         - name: nri-flex
+           config:
+             name: nextcloudFlex
+             global:
+               base_url: http://YOUR_HOST_IP/ocs/v2.php/apps/serverinfo/api/v1/
+               user: nextcloud_username
+               pass: nextcloud_password
+             apis:
+               - event_type: EVENT_TYPE
```
Comment (rhetoric101) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:51]:
```suggestion
               - event_type: YOUR_EVENT_TYPE
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx [hunk 2]
```diff
@@ -34,23 +34,23 @@ Complete the following steps to install the integration:
 
     1. Create a file named `nri-flex-nextcloud.yml` in `/etc/newrelic-infra/integrations.d`
 
-    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `EVENT_TYPE` and `YOUR_DOMAIN` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics on the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
+    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `EVENT_TYPE` and `YOUR_DOMAIN` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics in the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
```
Comment (rhetoric101) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx:37]:
```suggestion
    2. Use our [configuration template](https://github.com/newrelic/nri-flex/blob/master/examples/json-read-cmd-example.yml) to update the fields `YOUR_EVENT_TYPE` and `YOUR_HOST_IP` in the file named `nri-flex-nextcloud.yml`. The value on the `event_type` is used to store metrics in the NRDB. For example, you can update `EVENT_TYPE` to `NextcloudFlexMetrics`
```

**Review by rhetoric101 — COMMENTED** [review]
@sjyothi54 thanks for adding this follow-up PR! I just had a few minor suggestions.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx (hunk 2 above)
Comment (sjyothi54) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration.mdx]: @rhetoric101 You are correct. Updated the changes

**Review by jeff-colucci — APPROVED** [review]

**Pull request merged**